HTML/QuickForm/file.php

```diff
@@ -187,7 +187,7 @@ function onQuickFormEvent($event, $arg, &$caller)
                 break;
             case 'createElement':
                 $className = get_class($this);
-                $this->$className($arg[0], $arg[1], $arg[2]);

```

Comment:
That old code is somewhat scary construction.

Just to be safe, I grepped for other cases that might be like this (`$this->$foo`). None of the matches looked like obvious problems.